lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -317,13 +317,14 @@
 .forum-nav-thread-wrapper-1 {
     @extend %forum-nav-thread-wrapper;
     margin: 0 ($baseline / 4);
-    width: 80%;
+    max-width: calc(100% - 125px);
```

Comment:
What is this calculation doing? I don't like the hard-coded 125px. If it is necessary then it probably deserves a comment.